web/src/i18n/locales/ja.json

```
@@ -9,6 +9,10 @@
     " 吗？": "に変更しますか？",
     " 秒": " 秒",
     " 秒。": " 秒。",
+    "，当前无生效订阅，将自动使用钱包": "、有効なサブスクリプションがないため、自動的にウォレットを使用します",
```

Biome (2.4.10)

[error] The key ，当前无生效订阅，将自动使用钱包 was already declared.

(lint/suspicious/noDuplicateObjectKeys)